boot/mcuboot/Kconfig

```diff
@@ -14,7 +14,7 @@ if BOOT_MCUBOOT
 
 config MCUBOOT_VERSION
 	string "MCUboot version"
-	default "bb927ac014787992f2d8b42c7bd5d05eb4c17e05"
+	default "e4fba687d98bf44a71a7f37f1ef4637a662791c3"
```

Comment:
@xiaoxiang781216 it seems you’ve missed to push the commit hash change to 36bac4f6a54a3b76266444d8c6c177b8b0483a05

Reply:
Yes, you are right, fixed.